### PR TITLE
[MIG] stock_move_assign_picking_hook: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**

### CODE REVIEW

File: stock_move_assign_picking_hook/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{
+    "name": "Stock Move picking assignment hook",
+    "Summary": "Base module that adds an hook to override picking assignment on moves.",
+    "version": "14.0.1.1.0",
+    "development_status": "Alpha",
+    "author": "Camptocamp, BCIM, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "category": "Warehouse Management",
+    "depends": ["stock"],
+    "data": [],
```
Comment (rousseldenis) [inline stock_move_assign_picking_hook/__manifest__.py]: Please remove void entries

File: stock_move_assign_picking_hook/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Move picking assignment hook",
+    "Summary": "Base module that adds an hook to override picking assignment on moves.",
+    "version": "14.0.1.1.0",
+    "development_status": "Alpha",
```
Comment (rousseldenis) [inline stock_move_assign_picking_hook/__manifest__.py]: @jgrandguillaume Should the 14.0 modules stay in Alpha ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Minor comments

**Review by dsolanki-initos — APPROVED** [review]
LGTM

**Comment by dsolanki-initos** [comment]
Hi @phuctranerp can you please have a look at the minor changes so that we can merge this PR soon?

**Comment by phuctranfxvn** [comment]
Hi @dsolanki-initos

Thanks for reminding me, I have updated the PR.

File: stock_move_assign_picking_hook/__manifest__.py (hunk 2 above)
Comment (phuctranfxvn) [reply stock_move_assign_picking_hook/__manifest__.py]: Hello, can you help me recommend the correct development status of this module ? I am considering the module as stable. Let me know, thanks

**Comment by dsolanki-initos** [comment]
@rousseldenis It would be great if you can merge this PR.

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**